aptos-move/framework/aptos-framework/sources/configs/version.spec.move

```diff
@@ -1,8 +1,38 @@
 spec aptos_framework::version {
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+    }
+
     spec set_version {
         use aptos_framework::chain_status;
         use aptos_framework::timestamp;
+
         requires chain_status::is_operating();
         requires timestamp::spec_now_microseconds() >= reconfiguration::last_reconfiguration_time();
     }
+
+    spec set_version(account: &signer, major: u64) {
+        use std::signer;
+
+        aborts_if !exists<SetVersionCapability>(signer::address_of(account));
+        aborts_if !exists<Version>(@aptos_framework);
+
+        let old_major = global<Version>(@aptos_framework).major;
+        aborts_if !(old_major < major);
+    }
+
+    /// Abort if resource already exists in `@aptos_framwork` when initializing.
+    spec initialize(aptos_framework: &signer, initial_version: u64) {
+        use std::signer;
+
+        aborts_if signer::address_of(aptos_framework) != @aptos_framework;
+        aborts_if exists<Version>(@aptos_framework);
+        aborts_if exists<SetVersionCapability>(@aptos_framework);
+    }
+
+    /// Only for test
```

Comment:
Test functions don't need to be specified. Could you remove this spec?

Reply:
we add spec for `initialize_for_test` and set `verify = false`, because we opened `aborts_if_is_strict` in spec module, if we remove spec for `initialize_for_test`, prover will not let it pass. 
```
spec aptos_framework::version {
    spec module {
        pragma verify = true;
        pragma aborts_if_is_strict;
    }
```

Two solutions from us:
1 keep the spec and add a comment like "no need to add spec for test functions, just because the module turns on aborts_if_is_strict".(which we preferred)
2 remove the spec, and turn aborts_if_is_strict to false.

Reply:
OK. The current way looks fine.